bundles/org.openhab.ui/web/src/pages/settings/things/inbox/inbox-list.vue

```diff
@@ -23,6 +23,9 @@
       <f7-link color="orange" v-show="selectedItems.length" v-if="!$theme.md" class="ignore" @click="confirmActionOnSelection('ignore')" icon-ios="f7:eye_slash" icon-aurora="f7:eye_slash">
         &nbsp;Ignore {{ selectedItems.length }}
       </f7-link>
+      <f7-link color="blue" v-show="selectedItems.length" v-if="!$theme.md" class="unignore" @click="confirmActionOnSelection('unignore')" icon-ios="f7:eye" icon-aurora="f7:eye">
```

Comment:
I'm not sure if there's enough room on a phone screen for 4 links at the bottom - that's why I left the unignore feature out. Unignore could however replace ignore if all checked items are already ignored.

Reply:
![image](https://user-images.githubusercontent.com/23188584/121562619-ed59a380-ca19-11eb-8e5e-39d999273097.png)
On my phone (Android10) there is enough space because the Text is not shown.
But this might be depending on the phone or theme.

When starting, the absence of an 'unignore'-button is not cruical, but over time one may add more and more things to the ignore-list, and then it makes much more sense to have 'unignore' in the multiselect-view, too.

Reply:
Yep but for instance on a iPhone 12 mini (which would have the iOS theme and a smallish screen), it would look like this:

![image](https://user-images.githubusercontent.com/2004147/121578007-7aa4f400-ca2a-11eb-85ef-80312f077946.png)

this part of the code is for themes other than Material Design (i.e. Android) so adding a new function needs to account for those as well.